### PR TITLE
Change TCP to UDP

### PR DESCRIPTION
**Pull request by yotam180**

### CODE REVIEW

**Review by shaiTheKimhi — CHANGES_REQUESTED** [review]
Please specify what is the type of each parameter such as in line "def __init__(self, _type, status):"
and add documentation

**Comment by yotam180** [comment]
Shai, please point to a single function where there is no type-specification.

File: Communication/satprot.py [hunk 1]
```diff
@@ -1,72 +1,115 @@
...
 class OutMsg(object):
-    def __init__(self, _type, status):
+    """
+    Constructs messages for sending, and converting message objects to byte arrays.
+    """
+    def __init__(self, _type : int, status : int):
+        """
+        Initializes a new OutMsg object.
+        Parameters:
+            type - the type of the message (single byte int)
```
Comment (shaiTheKimhi) [inline Communication/satprot.py]: Can you elaborate?

**Review by shaiTheKimhi — COMMENTED** [review]
Left a comment.

**Review by shaiTheKimhi — COMMENTED** [review]
Add test in Tests directory instead of a test.py in Communication directory as the only file you will be abled to run in test.py in sat_monitoring_system directory

**Review by shaiTheKimhi — APPROVED** [review]

**Pull request merged**